src/Hng.Application/Features/SuperAdmin/Mappers/AdminUsersMappingProfile.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Hng.Application.Features.SuperAdmin.Dto;
+using Hng.Domain.Entities;
+
+namespace Hng.Application.Features.SuperAdmin.Mappers
+{
+    public class AdminUsersMappingProfile : AutoMapper.Profile
```

Comment:
what is the difference between this and the UserMappingProfile that already exists? It serves the same purpose.

Reply:
The userdto in user management contains fields that are not required on the endpoint like user organisations, products, profile etc, the admin user to requires the created_on property which the user management dto does not have. Extending an existing DTO may alter the behaviour or response of another endpoint

Reply:
> The userdto in user management contains fields that are not required on the endpoint like user organisations, products, profile etc, the admin user to requires the created_on property which the user management dto does not have. Extending an existing DTO may alter the behaviour or response of another endpoint

Okay, makes sense to me.